yesod-core/Yesod/Core/Class/Yesod.hs

```diff
@@ -366,6 +366,22 @@ sslOnlySessions = (fmap . fmap) secureSessionCookies
     setSecureBit cookie = cookie { setCookieSecure = True }
     secureSessionCookies = customizeSessionCookies setSecureBit
 
+-- | Helps defend against CSRF attacks by setting the SameSite attribute on
+-- session cookies to "Lax".
+laxSameSiteSessions :: IO (Maybe SessionBackend) -> IO (Maybe SessionBackend)
+laxSameSiteSessions = sameSiteSession sameSiteLax
+
+-- | Helps defend against CSRF attacks by setting the SameSite attribute on
+-- session cookies to "Strict".
```

Comment:
I've been thinking - maybe this functionality should be a plugin as opposed to in core? I think this because the spec is not quite stable yet (there was some ambiguity in this [rack](https://github.com/rack/rack/pull/1051) discussion). However for a plugin to be possible we must bump the Yesod `cookie` version, which I could do in another PR if that sounds good?